content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,55 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of its data requirements are missing from `dvc.yaml`. Namely, one of:
+
+- Files or directories in the <abbr>workspace</abbr> that are dependencies of
+  the stage are missing from `deps`.
+
```

Comment:
I thought I removed this line before. Is the formatting inserting it, or is that you @imhardikj? (Please see https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown about bullet lists.) 
```suggestion
```

Reply:
Actually it's my mistake. I but didn't notice that you removed the empty line in f578ddd06f32b6b660a5ff5917658d771ca76474. Updated now.

Reply:
No problem. Thanks